modules/vorbis/resource_importer_ogg_vorbis.cpp

```diff
@@ -116,135 +115,22 @@ Error ResourceImporterOggVorbis::import(ResourceUID::ID p_source_id, const Strin
 	return ResourceSaver::save(ogg_vorbis_stream, p_save_path + ".oggvorbisstr");
 }
 
-ResourceImporterOggVorbis::ResourceImporterOggVorbis() {
+#ifndef DISABLE_DEPRECATED
+Ref<AudioStreamOggVorbis> ResourceImporterOggVorbis::load_from_buffer(const Vector<uint8_t> &p_stream_data) {
+	return AudioStreamOggVorbis::load_from_buffer(p_stream_data);
 }
 
-void ResourceImporterOggVorbis::_bind_methods() {
-	ClassDB::bind_static_method("ResourceImporterOggVorbis", D_METHOD("load_from_buffer", "buffer"), &ResourceImporterOggVorbis::load_from_buffer);
-	ClassDB::bind_static_method("ResourceImporterOggVorbis", D_METHOD("load_from_file", "path"), &ResourceImporterOggVorbis::load_from_file);
+Ref<AudioStreamOggVorbis> ResourceImporterOggVorbis::load_from_file(const String &p_path) {
+	return AudioStreamOggVorbis::load_from_file(p_path);
 }
+#endif
 
-Ref<AudioStreamOggVorbis> ResourceImporterOggVorbis::load_from_buffer(const Vector<uint8_t> &file_data) {
-	Ref<AudioStreamOggVorbis> ogg_vorbis_stream;
-	ogg_vorbis_stream.instantiate();
-
-	Ref<OggPacketSequence> ogg_packet_sequence;
-	ogg_packet_sequence.instantiate();
```

Comment:
This was moved as is to `AudioStreamOggVorbis::load_from_buffer` (with just the parameter renamed to `p_stream_data` as it doesn't necessarily come from an actual file on disk).